@xen-orchestra/rest-api/src/groups/group.controller.mts

```diff
@@ -0,0 +1,51 @@
+import { Example, Get, Path, Query, Request, Response, Route, Security, Tags } from 'tsoa'
+import type { Request as ExRequest } from 'express'
+import { provide } from 'inversify-binding-decorators'
+import type { XoGroup } from '@vates/types'
+
+import { notFoundResp, unauthorizedResp, type Unbrand } from '../open-api/common/response.common.mjs'
+import { group, groupIds, partialGroups } from '../open-api/oa-examples/group.oa-example.mjs'
+import type { WithHref } from '../helpers/helper.type.mjs'
+import { XoController } from '../abstract-classes/xo-controller.mjs'
+
+@Route('groups')
+@Security('*')
+@Response(unauthorizedResp.status, unauthorizedResp.description)
+@Tags('groups')
+@provide(GroupController)
+export class GroupController extends XoController<XoGroup> {
+  // --- abstract methods
+  getAllCollectionObjects(): Promise<XoGroup[]> {
+    return this.restApi.xoApp.getAllGroups()
+  }
+  getCollectionObject(id: XoGroup['id']): Promise<XoGroup> {
+    return this.restApi.xoApp.getGroup(id)
+  }
+
+  /**
+   * @example fields "name,id,users:!empty"
+   * @example filter "users:!empty"
```

Comment:
Not a valid filter and can't work.
Please refer to the [documentation](https://docs.xen-orchestra.com/manage_infrastructure#number-comparison) to filter on the `users` property.